docs/plans/plugins/HOST-DATA-API.proto

```
@@ -1,10 +1,13 @@
-// PROPOSED — kandev.plugin.v1 Host data API (draft for review, not wired yet).
+// kandev.plugin.v1 Host data API — folded into
```

Buf (1.71.0)

[error] Filename "HOST-DATA-API.proto" should be lower_snake_case.proto, such as "host_data_api.proto".

(FILE_LOWER_SNAKE_CASE)